### PR TITLE
[Fixes #656] 'dotnet run' does not work on MusicStore app

### LINKED ISSUE #656: 'dotnet run' does not work on MusicStore app
Repro: do a `dotnet run` and this complains about unable to find `config.json` file

This happens in both release and dev branches. Seems like due to the recent project.json changes.

### PR DESCRIPTION
**Pull request by kichalla**
@Eilon @pranavkm

### CODE REVIEW

File: src/MusicStore.Standalone/project.json [hunk 1]
```diff
@@ -5,23 +5,34 @@
...
+        "../MusicStore/Program.cs",
+        "../MusicStore/bin",
+        "../MusicStore/obj"
+      ],
+      "include": [
+        "../../shared/**/*.cs",
+        "../MusicStore/**/*.cs"
+      ]
+    },
+    "copyToOutput": {
```
Comment (kichalla) [inline src/MusicStore.Standalone/project.json]: This was the main change that was required. Other changes are just to comply to the new project.json schema.

**Comment by kichalla** [comment]
Oops. This PR was supposed to be based on `release`. I will make sure to merge into `release` and then merge back to `dev`

File: src/MusicStore/web.config [hunk 2]
```diff
@@ -4,6 +4,6 @@
     <handlers>
       <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified" />
     </handlers>
-    <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" forwardWindowsAuthToken="true"/>
+    <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" forwardWindowsAuthToken="false" />
```
Comment (kichalla) [inline src/MusicStore/web.config]: This change was made by VS when I launched IIS express. Will revert.

**Comment by Eilon** [comment]
:shipit:

**Comment by kichalla** [comment]
9d3f2c4803125e438f8f152f1a4c7e88ee5c1ad0

**Pull request closed without merging**